lib/matplotlib/axes/_base.py

```diff
@@ -3283,7 +3284,9 @@ def ticklabel_format(self, *, axis='both', style='', scilimits=None,
             except (ValueError, TypeError) as err:
                 raise ValueError("scilimits must be a sequence of 2 integers"
                                  ) from err
-        STYLES = {'sci': True, 'scientific': True, 'plain': False, '': None}
+        STYLES = {'sci': True, 'scientific': True, 'plain': False, '': None, None: None}
+        # 'sci' is equivalent to 'scientific'.
```

Comment:
I think the fact that 'sci' and ‘scientific' are equivalent _should_ be in the docstring, as this is useful information for the user.

Reply:
Just put them back there! As for the pyplot issue, I attempted following the instructions and it told me, in my system, that the file got reformatted. Let's see how it goes!

Reply:
Yeah! I tried running the command again, but the file remained unchanged. I tried to change the file by hand, although this is not encouraged, and the commit went through on my fork, but it won't open here on Github. Just goes straight to a 404 page. Weird.

Here is the final change I made: https://github.com/pedrompecanha/matplotlib/commit/3cf0234d3646ef57264b8e19ddbaa6cd7d301bf7

Reply:
I believe it has gone through now. Must have been a Github instability. The tests seem to be running well, although I'm kind of scared of having to change this file by hand. Thoughts?

Reply:
Your manual change matches what I get by running the script, so I think it's OK.